README.md

```diff
@@ -44,9 +44,8 @@ Options:
 
 ### Sample Output
 
-```bash
 Generating Pip Tree Report...
-
+```json
```

Comment:
Can we instead just swap from `bash` to `text`? I'd accept that (keeping the backticks where they were previously). If we move the backticks, we fragment the output which would like a little awkward with JSON in formatting and the rest of the associated text output outside that formatting. May be better just to call it all "text" since that's the output of the tool?

```suggestion
```text
```

Reply:
yes, plain could be but you closed the PR so not sure what is your intention

Reply:
fixed